app/src/main/java/com/github/libretube/ui/sheets/EditChannelGroupSheet.kt

```diff
@@ -78,8 +84,23 @@ class EditChannelGroupSheet(
             group
         ) {
             group = it
+            updateConfirmStatus()
         }
         binding.subscriptionsContainer.isVisible = true
         binding.progress.isVisible = false
     }
+
+    private fun updateConfirmStatus() {
+        with(binding) {
+            val isGroupNameBlank = groupName.text?.isBlank() == true
+
+            groupName.error = if (isGroupNameBlank) {
+                requireContext().getString(R.string.group_name_error)
+            } else {
+                null
+            }
+
+            confirm.isEnabled = !isGroupNameBlank && group.channels.isNotEmpty()
```

Comment:
The confirm button is activates when the group name is not empty - whereas here, we additionally require that there's at least one channel selected. Both should behave the same way. I think we should also check if no existing group has the same name - if it's a new group - so that users don't accidentally override the old group.

Reply:
> whereas here, we additionally require that there's at least one channel selected

Yes, I added that requirement, because I don't think it should be possible to create empty groups, as they don't serve any purpose. I can revert the change if you want.

>  I think we should also check if no existing group has the same name - if it's a new group - so that users don't accidentally override the old group.

Done in 86854813e20ed71f9f52acfec7daa79fcc79aadd